components/ui/bottomSheet/BottomSheet.stories.tsx

```
@@ -0,0 +1,82 @@
+          type="button"
+          className="absolute left-1/2 top-1/2 -translate-x-1/2 -translate-y-1/2 rounded-full bg-primary-point px-6 py-3 typo-button text-white"
+          onClick={() => setIsOpen(true)}
+        >
+          긴 컨텐츠 바텀시트 열기
+        </button>
+        <BottomSheet isOpen={isOpen} toggleOpen={() => setIsOpen((prev) => !prev)}>
+          <div className="px-5 pb-8 pt-2">
+            <h2 className="typo-h2-sub mb-6 text-center text-primary-text">새 폴더 만들기</h2>
+            <div className="flex flex-col gap-6">
+              <div className="flex flex-col gap-2">
+                <label className="typo-body1 text-primary-text">폴더 이름</label>
```

Biome (2.4.6)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)